doc/reference/reference_lua/config.rst

```diff
@@ -524,6 +563,295 @@ The ``config.storage`` API allows you to interact with a Tarantool-based :ref:`c
+
+.. function:: connect(endpoints[, options])
+
+    Connect to a remote config.storage cluster.
+
+    :param table endpoints: table of config.storage instance URIs
+    :param table options: client options to pass. The following options are available:
+
+                           -   ``timeout`` (default: ``nil``, unset) -- connect & call timeout.
+
+    :return:    a <config.storage client> object
+
+    :rtype: table
+
+The example below shows how to connect to a specific config.storage cluster:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: config = require
+    :end-at: })
+    :dedent:
+
+The example below shows how to connect to a config storage cluster configured as the :ref:`centralized configuration storage <configuration_etcd>` in the :ref:`config.storage <configuration_reference_config_storage>` option:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: function connect_to_configured_storage
+    :end-at: config.storage_client.connect
+    :dedent:
+
+Examples on GitHub: `config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/config_storage>`_
+
+.. _config_storage_client_api_reference_put:
+
+.. function:: <config.storage client>:put(path, value)
+
+Put a value by the specified path to remote config.storage.
+
+:param string path: a path to put the value by
+:param string value: a value to put
+
+:return:    a table containing the following fields:
+
+            *   ``revision``: a revision after performing the operation
+
+:rtype: table
+
+**Example:**
+
+The example below shows how to read a configuration stored in the ``source.yaml`` file using the :ref:`fio module <fio-module>` API and put this configuration by the ``/myapp/config/all`` path:
+
+..  literalinclude:: /code_snippets/snippets/centralized_config/instances.enabled/config_storage/myapp.lua
+    :language: lua
+    :start-after: function put_config
+    :end-at: cluster_config_handle:close()
+    :dedent:
+
+Examples on GitHub: `config_storage <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/centralized_config/instances.enabled/config_storage>`_
+
+.. _config_storage_client_api_reference_get:
+
+.. function:: <config.storage client>:get(path)
+
+Get a value from remote config.storage stored by the specified path or prefix.
+
+:param string path: a path or prefix to get a value by; prefixes end with ``/``
+
+:return:    a table containing the following fields:
+
+            *   ``data``: a table containing the information about the value:
+
+                * ``path``: a path
+                * ``mod_revision``: the last revision at which this value was modified
+                * ``value:``: a value
+
+            *   ``revision``: a revision after performing the operation
```

Comment:
TBH, it is not very clear from this description, whether `data` contains the `{path = <...>, mod_revision = <...>, value = <...>}` mapping or an array of them. Since the call suport both point and prefix requests, I guess that it is either an array or a mapping of mappings. Anyway, I think that it would be nice to make it clear.

This is not really to this PR, because the description is the same as for the server API. Feel free to ignore.